src/Components/Blazor/Build/src/targets/All.props

```diff
@@ -1,8 +1,12 @@
 <Project>
   <Import Project="Blazor.MonoRuntime.props" />
 
+  <Import Project="StaticWebAssets.props" />
+
   <PropertyGroup>
-    <DefaultWebContentItemExcludes>$(DefaultWebContentItemExcludes);wwwroot\**</DefaultWebContentItemExcludes>
+
+    <!-- Assets for a Blazor app are exposed on the root folder by default. -->
+    <StaticWebAssetBasePath>/</StaticWebAssetBasePath>
```

Comment:
Would this result in conflicts if you reference two standalone apps from a hosted app?

Reply:
Yes, but that's expected.

I discussed this with @SteveSandersonMS and we agreed that it was ok in this case to require specifying the static web assets base path for one of them at that point to resolve the conflict.

Reply:
Just to follow up on this and for me to get a fuller understanding, when there's a conflict of this kind, what happens? Does the build fail? In the best case, I guess the error would tell you to use `StaticWebAssetBasePath`.

Reply:
The build fails telling you there are assets in conflicting paths.